indico/modules/events/papers/controllers/paper.py

```diff
@@ -167,13 +169,30 @@ def _process_assignment(self, assign):
             flash(_("Paper reviewing roles have been unassigned."), 'success')
         return jsonify_data(**self.list_generator.render_list())
 
+    def _get_conflicts(self, users):
+        conflicts = defaultdict(list)
+        for user in users:
+            if not user.affiliation:
+                continue
+            for contribution in self.contributions:
+                conflicts[user].extend(
```

Comment:
This always creates the entry in the dict, even if there are no conflicts. Which in principle is fine, except that you are checking `user in conflicts` in the template.  So you might want to use this in the template instead:

```
{% set user_conflicts = conflict.get(user) %}
{% if user_conflicts %}
...
```